tests/simple_server.py

```diff
@@ -73,6 +73,10 @@ def log_request(self, code='-', size='-'):
 # Allow re-use so you can re-run tests as often as you want even if the
 # tests re-use ports. Otherwise TCP TIME-WAIT prevents reuse for ~1 minute
 six.moves.socketserver.TCPServer.allow_reuse_address = True
-httpd = six.moves.socketserver.TCPServer(('', PORT), handler)
 
-httpd.serve_forever()
+try:
+  httpd = six.moves.socketserver.TCPServer(('', PORT), handler)
+  print("bind succeeded")
+  httpd.serve_forever()
+except:
+  print("bind failed")
```

Comment:
We don't really know it's the bind failing (and not something else)... but I think this is fine

Reply:
Yes, we don't care what exactly is the error within the try block.
The result is that the server doesn't work as expected and it's better to retry.